fdb-record-layer-core/src/main/proto/record_query_runtime.proto

```diff
@@ -22,34 +22,11 @@ syntax = "proto2";
 package com.apple.foundationdb.record.planprotos;
 import "google/protobuf/descriptor.proto";
 import "record_metadata.proto";
+import "record_query_plan.proto";
 
 option java_outer_classname = "RecordQueryRuntimeProto";
 option java_multiple_files = true;
 
-message PEnumLightValue {
```

Comment:
I am sure there is a reason, but why did this move back to the plan protos?

Reply:
because some "P*Value" depends on "PComparableObject"
if we keep "PComparableObject " in record_query_runtime.proto, then
record_query_plan depends on record_query_runtime.proto,
record_query_runtime depends on record_metadata.proto,
record_metadata.proto depends on record_query_plan (because Udf uses P*Value)
so there is a circular dependency.

so I moved all stuff related to "PComparableObject" into record_query_plan.proto